.travis.yml

```diff
@@ -4,6 +4,6 @@ rvm:
 before_install:
   - |
     if [ "$LUMOS" == "true" ]; then
-      GIST="https://gist.githubusercontent.com/camertron/c3386c9533e8ed32be35/raw/c212d6b6eb6ae463310d3ae187507a46f353b2af/txgh_lumos.patch"
+      GIST="https://gist.githubusercontent.com/camertron/c3386c9533e8ed32be35/raw/b399feaa1245a7bf89d3cc04ca44a42efb2ffdb5/txgh_lumos.patch"
```

Comment:
Not a big issue...but could you move your Gist config to a env var?  That way you won't need to checkin a change everytime you have a new Gist...also maybe I can swap over to using a patch type of approach for my config as well and we can drop the condition altogether